docs/weave-tutorial/06-buckets.md

```diff
@@ -0,0 +1,131 @@
+---
+id: buckets
+title: Buckets
+sidebar_label: Buckets
+---
+
+> [PR#8](https://github.com/iov-one/tutorial/pull/8): _Create buckets_
+\
+[PR#9](https://github.com/iov-one/tutorial/pull/9): _Add indexer for market using marketID, askTicker, bidTicker_
+
+When running your handlers, you get access to the root **KVStore**, which is an abstraction level similar to boltdb or leveldb. It is our data warehouse. An extension can opt-in to using one or more Buckets to store the data. Buckets offer the following advantages:
```

Comment:
I think this may require more clarification. 
What I think you are trying to explain here is that every handler when called (`Check` or `Deliver` methods) is given a database instance. This database represents a storage engine where all blockchain application state should be persisted.

Maybe it is better to say that bucket is like a document store for a single type of entity. It is using a database to put and retrieve data. It is a higher level API build on top of the low level (key-value) database.

I think it is not necessary to mention which database implementation we are using.

Reply:
```In Weave framework Buckets are the standard way to access and manipulate data, which is stored in **Key-Value Database*. Weave buckets could be found similar to [BoltDB](https://github.com/boltdb/bolt#using-buckets "Bolt Repo") or [LevelDB](https://github.com/google/leveldb "LevelDB Repo") design. Any extension can use one or multiple Buckets to store and access data. Buckets offer the following advantages:
```
I think this sentence serves the purpose. 

> I think it is not necessary to mention which database implementation we are using.

 Why not mention the underlying tech? I think giving implementation detail in between sentences will help doc readers to comprehend in detail.